docs/assets/css/style.css

```diff
@@ -35,6 +35,18 @@ html {
   -webkit-box-sizing: border-box;
   -moz-box-sizing: border-box;
   box-sizing: border-box; }
+  
+*::-webkit-scrollbar {
+  width: 15px;
```

Comment:
@ConX here right?
```suggestion
  width: 20px;
```

Reply:
Yeah, sure. Looks better this way!

Reply:
Agreed! @DhruvDutta if you agree with the suggestion. Please apply it by clicking on this button:
![image](https://github.com/devicons/devicon/assets/43886029/eb091077-5948-4176-a8f0-28fe4a8f28ae)